Jenkinsfile

```diff
@@ -56,9 +68,12 @@ stage('prep') {
       withCredentials([string(credentialsId: 'launchable-jenkins-bom', variable: 'LAUNCHABLE_TOKEN')]) {
         lines.each { line ->
           def commitHashes = readFile "commit-hashes-${line}.txt"
-          launchable("record build --name \"${BUILD_TAG}-${line}\" --no-commit-collection " + commitHashes)
-          launchable("record session --build \"${BUILD_TAG}-${line}\" --observation >launchable-session-${line}.txt")
-          stash name: "launchable-session-${line}.txt", includes: "launchable-session-${line}.txt"
+          launchable("record build --name ${env.BUILD_TAG}-${line} --no-commit-collection " + commitHashes + " --link \"View build in CI\"=${env.BUILD_URL}")
```

Comment:
Cosmetic fix: adding a link back to Jenkins to make it easier to view this information in the Launchable UI.